index.html

```diff
@@ -17,10 +17,17 @@
 			integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
 			crossorigin="anonymous"
 		/>
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```

Comment:
PRAISE: Nice job importing the Google Font using the link element. The other way(s) I can remember to import Google Fonts can make things ....non-non-heinous!